include/deal.II/grid/grid_out.h

```diff
@@ -865,7 +865,31 @@ namespace GridOutFlags
    * @ingroup output
    */
   struct Vtk : public DataOutBase::VtkFlags
-  {};
+  {
+    /**
+     * Default constructor.
+     */
+    Vtk()
```

Comment:
We typically write these constructors in a way so that they take the initializer values for the member variables as arguments, with default values `=true` in this case.